data.sql

```diff
@@ -16,3 +16,35 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Devimon', '2017-05-12', 11.0, true, 5);
 
+---- TASK 2-------
+UPDATE animals
+SET species = 'unspecified';
+UPDATE 7
```

Comment:
- [ ] Good job implementing the changes requested by the previous reviewer, :clap: as he mentioned Please ensure that your queries related to applying operations like `update`, `delete` etc should be inside the `queries.sql` not in the `data.sql` file. As I can see you added the same update command in the `queries.sql` too but still you missed the requirements of adding the command under the transactions. Which means you missed both of the requirements as I said. Kindly go through the requirements, and read the text carefully. :slightly_smiling_face: